### ISSUE TITLE
MDS download script & differently formatted studies

### ISSUE THREAD

**Issue by Woozl** [stagecc/bdc3-website-public#400]
The [MDS study API](https://gen3.biodatacatalyst.nhlbi.nih.gov/mds/metadata?_guid_type=discovery_metadata&limit=2000&data=true) returns studies mostly formatted like `phsXXXXXX.vX.pX.cX`, but also has several different entries: 

```
open_access-1000Genomes,
TOPMed_Common_Exchange_Area-Freeze_1,
TOPMed_Common_Exchange_Area-Freeze_10a,
TOPMed_Common_Exchange_Area-Freeze_8,
TOPMed_Common_Exchange_Area-Freeze_9b,
tutorial-biolincc_camp,
tutorial-biolincc_digitalis,
tutorial-biolincc_framingham,
tutorial-RECOVER_synthetic_data_set_1,
tutorial-synthetic_data_set_1
```

The download script that updates the files for the UI is written to only handle the first format:

https://github.com/stagecc/bdc3-website-public/blob/091341a6bf126bd5fd6fd8cd9218d36e2df45966/src/utils/studies-data-helpers.js#L55-L56C37

This issue is just to track the problem. _We're not sure if we need to support the other entries or if they're an error that can be filtered out._ Also note that the UI only displays a row in the table if there are a certain number of fields available. We first realized of this issue when `tutorial-RECOVER_synthetic_data_set_1` showed up on the UI -- it has enough fields populated to show up, which none of the other oddly formatted entries do.

@mbwatson is waiting on a response to determine out next steps.

**Comment by mbwatson** [comment]
i received some guidance today.

the PHS ID format is used for dbGaP studies. when there isn't a clear PHS ID from dbGaP, a Program-Project naming structure is used.

DRMWG is working on making sure these IDs are stored more clearly in Jira for internal tracking, but from our perspective, this is just a unique ID (string) that happens to follow patterns when possible for both dbGaP and non-dbGaP data,.

the ones whose keys start with `tutorial` are typically used for tutorials and are generally synthetic data. the others were a really old attempt at pulling in the exchange area. it's up to us (will confirm) if we want to pull those, but from Gen3's perspective, they are available datasets--just maybe not a dbGaP study.

**Comment by mbwatson** [comment]
Tim is going to check with DRMWG; we'll reconnect on Friday.

**Comment by mbwatson** [comment]
discussions are happening on the data side of things. they will consider (1) backfilling the missing data on their end and (2) dealing with this in the future.

however this also opened up a discussion of the overall utility of these studies tables in general. this will likely get folded into overall site redesign conversations, and i wouldn't be surprised if we continue to surface the same studies for a while.
